terraform/environments/bootstrap/delegate-access/iam.tf

```diff
@@ -27,7 +27,7 @@ module "ssm-cross-account-access" {
     aws = aws.workspace
   }
   account_id                  = local.environment_management.account_ids["core-shared-services-production"]
-  policy_arn                  = data.aws_iam_policy_document.execution-combined-policy.json
+  policy_arn                  = aws_iam_policy.execution-combined-policy.arn
```

Comment:
@davidkelliott I missed the arn here and supplied json... even though it said arn... not sure what I was thinking!